src/custom_source_buffers/abstract_source_buffer.ts

```diff
@@ -27,7 +27,7 @@ import ManualTimeRanges from "./time_ranges";
 interface IAbstractSourceBufferEvent { updatestart : undefined;
                                        update : undefined;
                                        updateend : undefined;
-                                       error : Event; }
+                                       error : any; }
```

Comment:
Does `unknown` works ?

Reply:
Nope. But I added a comment for that case :+1: